doc/whats_new/upcoming_changes/sklearn.mixture/30415.efficiency.rst

```diff
@@ -0,0 +1,5 @@
+- :func:`mixture.GaussianMixture` now consistently operates at `float32`
```

Comment:
Honestly I don't know whether there is a difference for sphinx but you may as well use `:class:` rather than `:func:`:
```suggestion
- :class:`~mixture.GaussianMixture` now consistently operates at `float32`
```